packages/graph/hash_graph/lib/graph/src/api/rest/middleware.rs

```diff
@@ -129,7 +148,7 @@ fn parse_x_forwarded_for(headers: &http::HeaderMap) -> Option<Cow<'_, str>> {
 }
 
 // Based on https://github.com/tokio-rs/axum/pull/769
-pub fn span_maker(request: &hyper::Request<hyper::Body>) -> tracing::Span {
+fn span_maker(request: &hyper::Request<hyper::Body>) -> tracing::Span {
```

Comment:
Also unnecessary
```suggestion
fn span_maker(request: &Request<Body>) -> tracing::Span {
```

Reply:
Thanks, done in 39700b961cfd014f3048ac96564a5a8c0837c584 :)